src/test/starknet.t.sol

```diff
@@ -105,7 +115,25 @@ contract StarknetTests is DssSpellTestBase, ConfigStarknet {
         _checkStarknetDaiBridge();
         _checkStarknetGovRelay();
         _checkStarknetCore();
-        _checkStarknetMessage(l2Spell);
+        _checkStarknetMessage(starknetValues.l2_spell);
```

Comment:
Strongly in favor of removing this check now that we are testing the event. It gives no extra value.

Reply:
It tests the message is actually transmitted. The event check does not.

Reply:
It checks that the message hash is added to an internal array (which happens in their contract right after the event emission). You don't know if that means anything is transmitted more than what the event tells you.
Since it makes us replicate some of their internal code and also makes us check the same thing twice I think we can do without it.
But not going to die on this hill though if you still want to leave it :)

If you decide to leave it I do suggest to move `_checkStarknetMessage` to the `testStarknetEventEmission` test (and renaming it, maybe to something like `testStarknetSpell`).
That way you have one test for static configuration and one for the spell. Plus you don't have to call `_payload()` twice (and can probably inline it).

Reply:
Seeing that `_checkStarknetMessage` was moved to `testStarknetSpell` I'll resolve this.